app/app.py

```diff
@@ -11,14 +11,14 @@
     langchain.debug = True
 
 from qna.llm import make_qna_chain, get_llm
-from qna.db import get_cache, get_vectorstore
+from qna.db import get_vectorstore#, get_cache
 from qna.prompt import basic_prompt
 from qna.data import get_arxiv_docs
 from qna.constants import REDIS_URL
 
-@st.cache_resource
-def fetch_llm_cache():
-    return get_cache()
+# @st.cache_resource
+# def fetch_llm_cache():
+#     return get_cache()
 
 @st.cache_resource
 def create_arxiv_index(topic_query, _num_papers, _prompt):
```

Comment:
should this commented out code be removed?

Reply:
I don't want to remove it because I will be adding caching back. It was [broken](https://redislabs.atlassian.net/browse/RAAE-415) when trying it due to some issue in the langchain class. So this will be a separate task/ticket to bring back the cache layer. But wanted to leave references in the code to help speed up the reimplementation.

Reply:
sounds good